utils/scripts/helpers/repo-helper-leap15.sh

```diff
@@ -37,22 +39,29 @@ disable_repos () {
 install_curl() {
 
     if command -v curl; then
+        echo "found curl!"
         return
     fi
 
-    if command -v dnf; then
-        dnf -y install curl
+    if command -v wget; then
+        echo "found wget!"
         return
     fi
 
-    if command -v zypper; then
-        zypper mr --all --disable
-        zypper addrepo                                                                           \
-            "${REPO_FILE_URL%/*/}/opensuse-proxy/distribution/leap/${BASE_DISTRO##*:}/repo/oss/" \
-              temp_opensuse_oss_proxy
-        zypper --non-interactive install curl
-        zypper removerepo temp_opensuse_oss_proxy
-    fi
+    # If we did not already have curl, we won't be able to easily install it.
+    #if command -v dnf; then
+    #    dnf -y install curl
+    #    return
+    #fi
+
+    #if command -v zypper; then
+    #    zypper mr --all --disable
+    #    zypper addrepo                                                                           \
+    #        "${REPO_FILE_URL%/*/}/opensuse-proxy/distribution/leap/${BASE_DISTRO##*:}/repo/oss/" \
+    #          temp_opensuse_oss_proxy
+    #    zypper --non-interactive install curl
+    #    zypper removerepo temp_opensuse_oss_proxy
+    #fi
```

Comment:
Should this just be removed?

Reply:
It could be in the future.   I missed deleting it for this pass.
This PR is one of the issues blocking getting CI running in the new lab.